src/component.ts

```diff
@@ -109,6 +109,13 @@ const reservedPropertyNames = [
   'filter'
 ]
 
+const shouldIgnore = {
+  'prototype': true,
+  'arguments': true,
+  'callee': true,
+  'caller': true
```

Comment:
Can we remove the quotes?

Reply:
Sure, quotes are removed, thank you for your kind advice!